src/api.raml

```diff
@@ -219,6 +219,16 @@ traits:
                   - created
                 required: false
                 default: updated
+            access:
+                description: |
+                    Return annotations appropriate for this access level.
+
+                    `restricted` will also return `public` annotations.
+                type: string
+                enum:
+                  - public
+                  - restricted
+                default: public
```

Comment:
The default is `public`, which means until journal passes this parameter explicitly, it will only see public annotations.

Since journal's API consumer is in the `view-restricted-annotations` group, it is up to journal to pass this parameter with a correct value; for example, hiding private annotations from the profile of A when it is B seeing it, but allowing private annotations when it is A itself seeing the page.